overlord/devicestate/devicestate.go

```diff
@@ -223,3 +225,44 @@ func ProxyStore(st *state.State) (*asserts.Store, error) {
 
 	return a.(*asserts.Store), nil
 }
+
+// plugConnected returns true if the given snap/plug names are connected
+func plugConnected(st *state.State, snapName, plugName string) bool {
```

Comment:
Is this really plugConnected/plugName or more like interfaceConnected/ifaceName?